<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>1028</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/down.png</normaloff>:/Image/down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Image/Luffy.png</normaloff>:/Image/Luffy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbtn_men">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>男</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_women">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>90</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbtn_dog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>dog</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_cat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>cat</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QCheckBox" name="cbx1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AAA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>BBB</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CCC</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>DDD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbx5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
